libcontainer/devices/devices_unix.go

```diff
@@ -94,6 +94,9 @@ func getDevices(path string) ([]*configs.Device, error) {
 			if err == ErrNotADevice {
 				continue
 			}
+			if os.IsNotExist(err) {
```

Comment:
Ok, I read it little closely and I am fine with your patch. getDevices() already ignores the ErrNotADevice error and continues and that should work.